query/src/main/java/com/connexta/replication/query/QueryService.java

```diff
@@ -170,12 +175,16 @@ private void doQuery() {
             .activeFiltersFor(site)
             .sorted(Comparator.comparingInt(Filter::getPriority).reversed())
             .collect(Collectors.toList());
+    LOGGER.debug("Filters retrieved for site {}:{}", site.getName(), filters);
     for (Filter filter : filters) {
       final FilterIndex filterIndex = queryServiceTools.getOrCreateFilterIndex(filter);
+      LOGGER.debug(
+          "Obtained FilterIndex {}, of filter {}, of site {}", filterIndex, filter, site.getName());
 
       // holds the latest modified time received so far
       AtomicReference<Instant> latestModifiedTime =
-          new AtomicReference<>(filterIndex.getModifiedSince().orElse(Instant.MIN));
+          new AtomicReference<>(
+              filterIndex.getModifiedSince().orElse(new Date(Long.MIN_VALUE).toInstant()));
```

Comment:
Solr can only save dates so using Instant.MIN causes an error. I started using the min date instead.

Reply:
Might be a good test to add to the Solr/spring-data integration test cases that deals with `Instant`